### DISCUSSION TITLE
HDMI output not working

### DISCUSSION THREAD

**Discussion by pawan2joy** [mendax1234/ThinkpadX390-Opencore-EFI#21] [General]
have the same thinkpad x390 with i7 10th gen, UHD 620.. HDMI output is not working, can you please help if i share my config file.
[config.zip](https://github.com/user-attachments/files/17810199/config.zip)

**Comment by mendax1234** [comment]
Hi, I am sorry that I no longer use Thinkpad X390 anymore. Can you post your question on the hackintosh forum like tonymacx86 etc. Sry
